lte/gateway/c/core/oai/lib/3gpp/3gpp_24.008.h

```diff
@@ -960,6 +961,32 @@ int decode_voice_domain_preference_and_ue_usage_setting(
 }
 #endif
 
+//------------------------------------------------------------------------------
+// 10.5.5.32 Extended DRX parameter
+//------------------------------------------------------------------------------
+#define EDRX_PARAMETER_IE_TYPE 3
+#define EDRX_PARAMETER_IE_MIN_LENGTH 3
+#define EDRX_PARAMETER_IE_MAX_LENGTH 3
+
+typedef struct edrx_parameter_s {
+  uint8_t length;
+  uint8_t pagingtimewindow : 4;
+  uint8_t edrxvalue : 4;
+} edrx_parameter_t;
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+int encode_edrx_parameter_ie(edrx_parameter_t* edrxparameter,
```

Comment:
This function is declared but never defined. Could you please add a definition for this too?